src/pages/networks/actions/DeleteNetworkBtn.tsx

```diff
@@ -49,7 +49,7 @@ const DeleteNetworkBtn: FC<Props> = ({ network, project }) => {
           ? "Can not delete, network is not managed"
           : isUsed
             ? "Can not delete, network is currently in use"
-            : "Delete network"
+            : "This network can be deleted"
```

Comment:
I think this can just be empty. No need to highlight, that we can delete it, when the button is active.

Reply:
Nice! so empty string for the title attribute won't trigger hover text, thanks for the tip!